semperfiwebdesign/all-in-one-seo-pack
# Add Preview Snippet & Character Counter support for text tab of Classic Editor

Issue #3052

## Proposed changes

Adds support for the Preview Snippet and Character Counter for the text tab of the Classic Editor.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Improves existing functionality
- Improves existing code

## Checklist


## Testing instructions

1. Reproduce the issue here - https://github.com/semperfiwebdesign/all-in-one-seo-pack/issues/3052
2. Install the PR and confirm that it fixes the issue.
3. Do a quick check of the Preview Snippet and Character Counter for the Edit post page (both for the Gutenberg Editor and Classic Editor). 
4. Do a quick check of the Character Counter for the Home Title and Home Description fields in the General Settings and Social Meta menu.

## Further comments

Grunt will fail on this PR and that's fine.

I've also deleted the relevant minified files for now since we're not using them at the moment and now are out-of-date.

Refactored some of the code too and created a shared admin JS file for general functions.
